### PR TITLE
Updated the debug controller presentation style to make it look less funny on iPads.

### PR DESCRIPTION
**Pull request by htinlinn**
@Haud @MichaelCampbell I gave a shot implementing the clear window technique we discussed. Check it out; it seems to work pretty well.

### CODE REVIEW

File: Yoshi/Yoshi/YoshiConfigurationManager.swift [hunk 1]
```diff
@@ -34,39 +34,39 @@ internal final class YoshiConfigurationManager {
...
-        window.rootViewController?.view.layoutIfNeeded()
-        animate {
-            window.alpha = 1
+        
+        presentingWindow = window
+        
+        // iOS doesn't like when modals are presented right away after a window's been made key and visible.
+        // We need to delay presenting the debug controller a bit to suppress the warning.
+        dispatch_after(dispatch_time(DISPATCH_TIME_NOW, Int64(0 * Double(NSEC_PER_SEC))), dispatch_get_main_queue()) { [weak self] in
+            self?.presentDebugViewController()
```
Comment (Haud) [inline Yoshi/Yoshi/YoshiConfigurationManager.swift]: Do we need weak here? I feel like this block will be destroyed after it is executed (also it's a singleton), so we can be confident that it'll exist to handle this.

**Comment by Haud** [comment]
One small comment, but this works awesome. I think this fixes @MichaelCampbell 's problem.

File: Yoshi/Yoshi/YoshiConfigurationManager.swift (hunk 1 above)
Comment (htinlinn) [inline Yoshi/Yoshi/YoshiConfigurationManager.swift]: @Haud oh yeah. that makes sense. removed.

**Comment by Haud** [comment]
:+1:

**Comment by MichaelCampbell** [comment]
:+1:

**Comment by MichaelCampbell** [comment]
:+1:

**Pull request merged**